src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -354,4 +355,13 @@ public void saveReDetermination(
     public void updateClaimState(String authorisation, Claim claim, ClaimState state) {
         caseRepository.updateClaimState(authorisation, claim.getId(), state);
     }
+
+    public void updateClaimSubmissionOperationIndicators(String authorisation,
+                                         Claim claim,
+                                         ClaimSubmissionOperationIndicators claimSubmissionOperationIndicators) {
+        caseRepository.updateClaimSubmissionOperationStatus(authorisation,
```

Comment:
formatting here too - just be consistent with class and put a case forward for change 😸

Reply:
done.